frontend/package.json

```diff
@@ -101,6 +100,7 @@
     "@babel/preset-env": "^7.24.5",
     "@babel/preset-react": "^7.24.1",
     "@babel/preset-typescript": "7.18.6",
+    "@kubernetes/client-node": "^1.0.0-rc4",
```

Comment:
Changed this to be a dev dependency because we are only using type definitions from it.